test/unit/provenance/test_rocrate.py

```diff
@@ -504,114 +503,6 @@ def test_rocrate_main_fail_missing_rocrate(
-                'EXPID': _EXPID
-            },
-            'EXPERIMENT': {},
-            'CONFIG': {},
-            'ROOTDIR': str(experiment_path),
-            'PROJECT': {
-                'PROJECT_DESTINATION': '',
-                'PROJECT_TYPE': 'LOCAL'
-            },
-            'LOCAL': {
-                'PROJECT_PATH': str(project_path)
-            },
-            'APP': {
-                'INPUT_1': 1,
-                'INPUT_2': 2
-            },
-            'ROCRATE': {
-                'INPUTS': ['APP'],
-                'OUTPUTS': [
-                    'graph_*.gif'
-                ],
-                'PATCH': json.dumps({
-                    '@graph': [
-                        {
-                            '@id': './',
-                            "license": "Apache-2.0"
-                        }
-                    ]
-                })
-            }
-        }
-        create_conf_dir(experiment_path, as_conf=mocked_as_conf)
-        mocked_as_conf.get_storage_type.return_value = 'pkl'
-        mocked_as_conf.get_date_list.return_value = []
-
-        mocked_get_autosubmit_version.return_value = '4.0.0b0'
-        mocked_get_experiment_descrip.return_value = [
-            ['mocked test project']
-        ]
-
-        mocked_job_list = Mock()
-        mocked_JobList.return_value = mocked_job_list
-
-        job1 = Mock(autospec=Job)
-        job1_submit_time = datetime.datetime.strptime("21/11/06 16:30", "%d/%m/%y %H:%M")
-        job1_start_time = datetime.datetime.strptime("21/11/06 16:40", "%d/%m/%y %H:%M")
-        job1_finished_time = datetime.datetime.strptime("21/11/06 16:50", "%d/%m/%y %H:%M")
-        job1.get_last_retrials.return_value = [
-            [job1_submit_time, job1_start_time, job1_finished_time, 'COMPLETED']]
-        job1.name = 'job1'
-        job1.date = '2006'
-        job1.member = 'fc0'
-        job1.section = 'JOB'
-        job1.chunk = '1'
-        job1.processors = '1'
-
-        job2 = Mock(autospec=Job)
-        job2_submit_time = datetime.datetime.strptime("21/11/06 16:40", "%d/%m/%y %H:%M")
-        job2_start_time = datetime.datetime.strptime("21/11/06 16:50", "%d/%m/%y %H:%M")
-        job2_finished_time = datetime.datetime.strptime("21/11/06 17:00", "%d/%m/%y %H:%M")
-        job2.get_last_retrials.return_value = [
-            [job2_submit_time, job2_start_time, job2_finished_time, 'COMPLETED']]
-        job2.name = 'job2'
-        job2.date = '2006'
-        job2.member = 'fc1'
-        job2.section = 'JOB'
-        job2.chunk = '1'
-        job2.processors = '1'
-
-        mocked_job_list.get_job_list.return_value = [job1, job2]
-        mocked_job_list.get_ready.return_value = []  # Mock due the new addition in the job_list.load()
-        mocked_job_list.get_waiting.return_value = []  # Mocked due the new addition in the job_list.load()
-        autosubmit = Autosubmit()
-        r = autosubmit.rocrate(_EXPID, path=Path(temp_dir))
-        assert r
-
-
```

Comment:
Move to an integration test, so we could use `autosubmit_exp` and remove all the mock calls (this test failed as the code and the mocks were not matching -- the danger of mocking too much).